app/src/main/res/values-ru-rRU/strings.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ * Tinc App, an Android binding and user interface for the tinc mesh VPN daemon
+ * Copyright (C) 2017-2023 Pacien TRAN-GIRARD
```

Comment:
You can add your name or nickname to the copyright of that file, if you wish so.

Reply:
Thank you. No need to mention me.